imgui.h

```diff
@@ -2742,6 +2743,8 @@ struct ImFont
     ImVector<float>             IndexAdvanceX;      // 12-16 // out //            // Sparse. Glyphs->AdvanceX in a directly indexable way (cache-friendly for CalcTextSize functions which only this this info, and are often bottleneck in large UI).
     float                       FallbackAdvanceX;   // 4     // out // = FallbackGlyph->AdvanceX
     float                       FontSize;           // 4     // in  //            // Height of characters/line, set during loading (don't change after loading)
+    float                       ExtraLineHeight;    // 4     // in  //            // Extra line height
+    float                       ExtraLineAdvance;   // 4     // in  //            // Extra spacing between lines
```

Comment:
I think the comment above should include these new fields.

Reply:
I added real comments here.

Reply:
The line right above describes how many bytes are hot for this struct.

https://github.com/ocornut/imgui/blob/630e42c47a513faaf83a0d5f3d0223857cfa7773/imgui.h#L2744

I don't know how `CalcTextSize()` works exactly, but if it would want to break lines, it would prompt inclusion of these new fields into the hot memory calculations.

Reply:
I moved new fields down. They're not needed in hot path nor reason to invalidate assumption made by this comment.

Thanks for pointing that out.